tests/test_synthetic_pii.py

```diff
@@ -469,7 +469,8 @@ def test_employer_name_map(mocker):
     assert not (employer_names["employer_name"].isnull().any())
 
 
-def test_ssn_mapping():
+@pytest.mark.slow
```

Comment:
To run this you'd now need to do `pytest --runslow`. This allows for quick unit tests as well as passing with CI (since CI does not runslow). It's a compromise.

Reply:
This is no longer necessary - the pytests now work quickly.